### ISSUE TITLE
Is it possible to have tasks or could it be added?

### ISSUE THREAD

**Issue by alexanderjohnen** [ssborbis/ContextSearch-web-ext#733]
We have a website at work that only finds the searched keyword if I am logged in.

So currently I have a shortcut to open the websites login (which thankfully automatically logs me in then because of some autofill plugin) and then I can search for the keyword. Said website doesnt keep you logged in for long (maybe 30 minutes idk) so I have to constantly repeat this process.

I am wondering if it's possible to have tasks within engines. Like, if I search for a keyword with ContextSearch, that it would open one website first, wait for the login, then open the link with my keyword search.

**Comment by ssborbis** [comment]
Yeah, you can probably use the post-search script field to run javascript commands that can do that.

**Comment by ssborbis** [comment]
I'll need to think on it. There may be a way, depending on how much of the webextension API is accessible from the content scripting. There's certainly nothing baked into this addon that will do it atm. I'll let you know here if I come up with something.
